avm/res/network/bastion-host/main.bicep

```
@@ -8,8 +8,8 @@ param name string
 @description('Optional. Location for all resources.')
 param location string = resourceGroup().location
 
-@description('Required. Shared services Virtual Network resource identifier.')
-param vNetId string
+@description('Required. Shared services Virtual Network resource Id.')
+param vNetResourceId string
```

At this point what about aligning with what in az fw 
```suggestion
param virtualNetworkResourceId string
```

Will not block the PR on this, just seemed strange not to mention, since we're updating the same parameter used by the same setting implementation

---

done